todo_app/templates/index.html

```diff
@@ -2,17 +2,43 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
+  <div class="jumbotron text-white" style="background-color: #53669f;">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">My to-do app.</p>
   </div>
 
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+        <li>ID: {{ item.id }} - {{ item.title }} - {{item.status}}</li>
+        {% endfor %}
       </ul>
     </div>
   </div>
-{% endblock %}
\ No newline at end of file
+  
+  <!-- added for add item -->
+    <div class="row">
+      <form class="col-auto" action="/additem" method="POST" id="newitem">
+        <div class="input-group">
+          <input name="title" type="text" class="form-control" placeholder="Enter new Item" required /><span>
+          <button type="submit" class="btn btn-primary"> Add Item </button></span>
+        </div>  
+      </form>
+    </div>  
+  <!-- Stretch Goals-->
+
+  <!--Add functionality to mark an item as completed-->
+    <div class="row">
```

Comment:
A suggestion, but you don't need to fix this necessarily:
Instead of having the user type in the id of the item and, consequently, having to display it above, you could:
* Move this logic inside the for loop above
* Add a button for each card that's not Completed
* Add the id field as a hidden input
* Now you won't need to display the card ID (which will not be as readable in next exercises)
e.g:

```
{% for item in items %}
    <div class="d-flex">
        <li>ID: {{ item.id }} - {{ item.title }} - {{item.status}}</li>
        {% if item.status != 'Complete' %}
            <form class="col-auto" action="/complete" method="POST" id="complete">
                <input name="ID" type="hidden" Class="form-control" value={{ item.id }} />
                <button type="submit"class="btn btn-success"> Completed </button>
            </form>
        {% endif %}
    </div>
{% endfor %}
```